tests/phpunit/tests/ajax/CropImage.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+/**
+ * Admin Ajax functions to be tested.
+ */
+require_once ABSPATH . 'wp-admin/includes/ajax-actions.php';
+
+require_once ABSPATH . 'wp-admin/includes/class-wp-filesystem-base.php';
+require_once ABSPATH . 'wp-admin/includes/class-wp-filesystem-direct.php';
+
+/**
+ * Class for testing ajax crop image functionality.
+ *
+ * @group ajax
+ */
+class Tests_Ajax_CropImage extends WP_Ajax_UnitTestCase {
```

Comment:
The class name should follow this convention `Tests_[group]_FunctionName`, which changes this test class to `Tests_Ajax_WpAjaxCropImage`.

and the file should be camelCase of the function name, e.g. `wpAjaxCropImage.php`.

Reply:
That's good to know! Thank you.
Fixed in 743688eec4566886bdfac2f9c0312e224eb19bc0

Reply:
>P.S. It seems that all classes/files in the `tests/phpunit/tests/ajax` folder have to be renamed because they don't follow this convention.

This is true throughout the test suite. Can be remedied as part of https://core.trac.wordpress.org/ticket/54725. Feel free to create a PR.